src/index.ts

```diff
@@ -603,6 +603,23 @@ function algoEncode(data: Buffer): string {
   return cleanAddr;
 }
 
+function hntAddresEncoder(data: Buffer): string {
+  const buf = Buffer.concat([Buffer.from([0]), data]);
+
+  return bs58Encode(buf);
+}
+
+function hntAddressDecoder(data: string): Buffer {
```

Comment:
Is this correct? Looks like it's doing checksum

https://github.com/helium/helium-js/blob/c221ad920b5c4c4fc5f01bf3fc3eee82cc52599c/packages/crypto-react-native/src/utils.ts#L43

You may want to try with the test used by their official spec at https://github.com/helium/helium-js/blob/master/packages/crypto/src/__tests__/Address.spec.ts#L8

Reply:
It's doing the checksum which already exists in the base58check. I've added ECC_COMPACT_ADDRESS as test case aswell, the other test case is from: https://github.com/helium/helium-js/blob/master/integration_tests/fixtures/users.ts#L39

I also tried their implementation just to be sure about it, and result is exactly the same.